_includes/seo.html

```diff
@@ -113,7 +115,7 @@
   <script type="application/ld+json">
     {
       "@context" : "http://schema.org",
-      "@type" : "{% if site.social.type %}{{ site.social.type }}{% else %}person{% endif %}",
+      "@type" : "{% if site.social.type %}{{ site.social.type }}{% else %}Person{% endif %}",
```

Comment:
Thanks for the background. I noticed it should be capitalized it after seeing feedback from https://webmaster.yandex.com/microtest.xml.